unlock-app/next.config.js

```diff
@@ -66,6 +68,48 @@ module.exports = withSourceMaps({
       '/terms': { page: '/terms' },
       '/privacy': { page: '/privacy' },
       '/log': { page: '/log' },
+      '/post': { page: '/post' },
+      '/blog': { page: '/blog' },
     }
+
+    // Find blog posts to export and render them as static pages (*.md files in the /blog folder)
+    let posts = fs.readdirSync(join(dir, 'static', 'blog'))
+    let postFeed = [] // We will use this to populate the homepage and (later) an RSS feed
+
+    // Ensure posts are sorted in reverse chronological order for the index
+    posts.sort(function(a, b) {
+      return (
+        fs.statSync(dir + b).mtime.getTime() -
```

Comment:
I worry that using the files' metadata may bite us? Could we use the `publishDate` inside the files themselves?